odmObject/odmobject-accelerometer.sdf.json

```diff
@@ -0,0 +1,60 @@
+{
+  "info": {
+    "title": "OMA LwM2M Accelerometer (Object ID 3313)",
+    "version": "20191002",
+    "copyright": "Copyright (c) 2018 IPSO.",
+    "license": "https://github.com/one-data-model/oneDM/blob/master/LICENSE"
+  },
+  "namespace": {
```

Comment:
i thought we decided that odm is the default name space..
e.g. this namespace declaration should not be there

Reply:
True, for ODM Objects it should not be there. For native IPSO objects it should be. Right now the files are kind of both. But I'll re-generate versions without.